tests/sdk/nodejs/examples/helm-release-crd/step2/index.ts

```diff
@@ -0,0 +1,15 @@
+import * as k8s from "@pulumi/kubernetes";
+
+const prometheusRelease = new k8s.helm.v3.Release("prometheus", {
+    name: "kube-prometheus-stack",
+    chart: "kube-prometheus-stack",
+    repositoryOpts: {
+        repo: "https://prometheus-community.github.io/helm-charts",
+    },
+    values: {
+        commonLabels: {
```

Comment:
Nit: add a comment somewhere about the purpose of this test (what step2 is verifying)

Reply:
Done